samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Content/Controls/DrawerFlyoutPresenterSamplePage.xaml

```diff
@@ -0,0 +1,93 @@
+﻿<Page x:Class="Uno.Toolkit.Samples.Content.Controls.DrawerFlyoutPresenterSamplePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:local="using:Uno.Toolkit.Samples.Content.Controls"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:toolkit="using:Uno.UI.Toolkit"
+	  xmlns:sample="using:Uno.Toolkit.Samples"
+	  xmlns:utu="using:Uno.Toolkit.UI"
+	  mc:Ignorable="d"
+	  Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+	<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+		<sample:SamplePageLayout x:Name="SamplePageLayout">
+			<sample:SamplePageLayout.MaterialTemplate>
```

Comment:
Should this maybe go within a new Fluent style section?

Should we also include an actual Material style for this? Maybe something similar to the Material Flyout styles that we already have in Themes

Reply:
Not sure about that, since this style is mostly about providing a raw functionality. It does practically nothing about the visual itself. You could define some default margin/border/background/padding and bundle it with the style, but it would be better in a different issue/PR?